### DISCUSSION TITLE
How can I lower the brightness on a video?

### DISCUSSION THREAD

**Discussion by jocelynasp** [bonsai-rx/bonsai#946] [Q&A]
I got a new camera for recording behavior (just recording, no tracking or anything just yet) and the light is so bright it's reflecting off the surfaces. Is there any way I can lower the brightness on the video?

![image](https://user-images.githubusercontent.com/99834393/175550859-5e19cf90-b0ce-4417-8fec-6e4a4617be12.png)

**Comment by bruno-f-cruz** [comment]
Hi,
I am not sure i fully understood the scope of the question. Once the frame is acquired, you wont be able to change the dynamic range of the input. You can do transformations on the data, like halving all values, but that wont fix your problem (especially if you are losing information due to saturation in the camera sensor).

If saturation is indeed the problem, you can always use less light or reduce your camera exposure. You can do this by setting the exposure property inside the `CameraCapture` node.

Hope it helps !

![image](https://user-images.githubusercontent.com/7049351/175830636-2c36a1d2-4566-4d1d-a80b-94dbcdb75ebd.png)
